### PR TITLE
#135 Ethereum mempool indexation pump

### PR DESCRIPTION
**Pull request by arturalbov**
Ethereum mempool indexation pump.
Also related to https://github.com/cybercongress/cyber-search/issues/121

### CODE REVIEW

File: pumps/ethereum/src/main/kotlin/fund/cyber/pump/ethereum/client/ParityToEthereumBundleConverter.kt [hunk 1]
```diff
@@ -22,46 +25,72 @@ class ParityToEthereumBundleConverter(
...
     }
 
+    fun parityMempoolTxToDao(parityTx: Transaction): EthereumTx {
+        return EthereumTx(
+            from = parityTx.from.toSearchHashFormat(), to = parityTx.to.toSearchHashFormat(),
+            nonce = parityTx.nonce.toLong(),
+            value = BigDecimal(parityTx.value) * weiToEthRate,
+            hash = parityTx.hash.toSearchHashFormat(), blockHash = null,
+            blockNumber = -1L,
+            blockTime = Instant.now(),
```
Comment (hleb-albau) [inline pumps/ethereum/src/main/kotlin/fund/cyber/pump/ethereum/client/ParityToEthereumBundleConverter.kt]: observedTime

File: common/src/main/kotlin/fund/cyber/search/configuration/Env.kt [hunk 2]
```diff
@@ -32,11 +32,13 @@ const val CORS_ALLOWED_ORIGINS = "CORS_ALLOWED_ORIGINS"
 const val CORS_ALLOWED_ORIGINS_DEFAULT = "search.fund.cyber.fund"
 
 const val START_BLOCK_NUMBER = "START_BLOCK_NUMBER"
-const val START_BLOCK_NUMBER_DEFAULT = -1L
+const val START_BLOCK_NUMBER_DEFAULT = -2L
```
Comment (hleb-albau) [inline common/src/main/kotlin/fund/cyber/search/configuration/Env.kt]: -2?

File: contract-summary/ethereum/src/main/kotlin/fund/cyber/contract/ethereum/summary/EthereumContractSummaryDelta.kt [hunk 3]
```diff
@@ -78,14 +78,14 @@ class EthereumTxDeltaProcessor : DeltaProcessor<EthereumTx, CqlEthereumContractS
                 contract = tx.from, txNumberDelta = 1, minedBlockNumberDelta = 0, uncleNumberDelta = 0,
                 balanceDelta = tx.value.negate() - tx.fee, totalReceivedDelta = BigDecimal.ZERO,
                 smartContract = (tx.createdSmartContract != null), topic = record.topic(),
-                partition = record.partition(), offset = record.offset(), time = tx.blockTime
+                partition = record.partition(), offset = record.offset(), time = tx.blockTime!!
```
Comment (hleb-albau) [inline contract-summary/ethereum/src/main/kotlin/fund/cyber/contract/ethereum/summary/EthereumContractSummaryDelta.kt]: do not forget!

**Comment by abitrolly** [comment]
Better reference upstream issue on PR merge with comment "Closes #135".

**Pull request merged**